src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -99,34 +99,33 @@ private async Task TriggerAnalysisAsync(Request request, IIssueConsumer consumer
             await TaskScheduler.Default;
 
             logger.WriteLine($"Analyzing {request.File}");
+            var messages = new List<Message>();
+            Action<Message> handleMessage = message => messages.Add(message); // todo: change to consumer.Accept
 
             // We're tying up a background thread waiting for out-of-process analysis. We could
             // change the process runner so it works asynchronously. Alternatively, we could change the
             // RequestAnalysis method to be synchronous, rather than fire-and-forget.
```

Comment:
Unrelated: I think I meant to say "asynchronous" in the comment on line 107, not "synchronous"...